particle-iot/device-os
# [ci] minor update sc-101315/device-os-manifest

### Problem

The manifest schema was updated with a new platform field (`baseMcu`) which caused the local `.workbench/manifest.json` overrides file to fail validation. 

### Solution

Removing the redundant entries avoids needlessly overriding default toolchain dependencies. In the future, only differences from the default manifest need to be specified in `.workbench/manifest.json`. for example, if we upgrade to a newer version of the `compilers` dependency, that [should be specified like this](https://github.com/particle-iot/device-os/commit/fb88c8f1336e8f51f767593ff06950de199d7b4e)

### Steps to Test

1. Review the [updated `manifest.json` file](https://github.com/particle-iot/device-os/blob/bug/sc-101315/device-os-manifest/.workbench/manifest.json)
5. Observe the [latest CI run in CircleCI](https://app.circleci.com/pipelines/github/particle-iot/device-os/667/workflows/6b79ae5f-144a-4e32-b4f5-fd51fceb4529)

**outcome**

CI workflow should pass, changes should be appropriate

### References

https://app.shortcut.com/particle/story/101315/device-os-test-build-system-ci-workflow-occasionally-errors#activity-102481
https://github.com/particle-iot/cli/pull/122

---

### Completeness
